initdb.py

```
@@ -222,7 +220,90 @@ def __create_manifest_and_tags(
         )
 
 
-def __generate_repository(user_obj, name, description, is_public, permissions, structure):
+def create_schema2_manifest_for_testing(repo, structure, tag_map):
+    """
+    Creates a Docker v2 schema 2 manifest with the given structure.
+    TODO: Consider reimporting the function after resolving persistency problem. Schema 2 validation
+    expects that config layer is available in storage, but storage doesn't appear to persist across
+    tests.
+    """
+    num_layers, subtrees, tag_names = structure
```

Ruff (0.15.4)

[warning] Unpacked variable `subtrees` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)